techmatters/flex-plugins
# WorkerDirectoryTabs: added hidden filter for available workers only

## Description
This PR adds a "hidden worker filter" to filter out the non-available counselors from the workers directory tab (transfer).

![Screenshot from 2023-03-28 18-19-14](https://user-images.githubusercontent.com/15805319/228304653-6f3d4334-15d5-44f9-b77b-c4a51ac3d371.png)

### Checklist
- [x] [Corresponding issue has been opened](https://tech-matters.atlassian.net/browse/CHI-1732)
- [n/a] Feature flags added
- [n/a] Strings are localized

### Verification steps
- Start local server (`npm run dev`).
- Start a new convo and accept the new task with a counselor A.
- Login to Flex with another counselor B and set it's status as `Available` (or ask someone in the team to do so).
- Confirm that only the `Available`(s) counselor(s) shows in the "workers directory" when attempting to transfer the task.
